input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -21,76 +25,76 @@ All view runners must implement these FHIRPath capabilities:
 * Math operators: [addition (+)](https://hl7.org/fhirpath/#addition), [subtraction (-)](https://hl7.org/fhirpath/#subtraction), [multiplication (*)](https://hl7.org/fhirpath/#multiplication), [division (/)](https://hl7.org/fhirpath/#division)
 * Comparison operators: [equals (=)](https://hl7.org/fhirpath/#equals), [not equals (!=)](https://hl7.org/fhirpath/#not-equals), [greater than (>)](https://hl7.org/fhirpath/#greater-than), [less or equal (<=)](https://hl7.org/fhirpath/#less-or-equal)
 
-<sup>*</sup> Not yet part of the normative FHIRPath release, currently in draft.
+<sup>*</sup> Not yet part of the normative [FHIRPath](https://hl7.org/fhirpath/) release, currently in draft.
+
+#### Optional Core FHIRPath Functions
+
+*View runners* are encouraged to implement these functions as well to support a
+broader set of use cases:
+
+* [memberOf](https://hl7.org/fhir/R4/fhirpath.html#functions) function
+* [toQuantity](https://hl7.org/fhirpath/#toquantityunit-string-quantity) function
 
-#### Additional functions
+#### Required Additional Functions
 
-All view runners must implement these functions that do not exist in the
-FHIRPath specification but are necessary in the context of defining views:
+All *view runners* must implement these functions that extend the
+FHIRPath specification. Despite not being in the [FHIRPath](https://hl7.org/fhirpath/) specification, they are necessary in the context of defining views:
 
-##### getResourceKey() : KeyType
+* [getResourceKey](https://hl7.org/fhir/R4/fhirpath.html#functions) function
+* [getReferenceKey](https://hl7.org/fhirpath/#toquantityunit-string-quantity) function
```

Comment:
These links are wrong, they should probably go to `#getresourcekey--keytype` and `#getreferencekeyresource-type-specifier--keytype` respectively.

Reply:
Good catch. Fixed!